File: bookingSystem/MyApp/src/app/model/property.ts
import { IPropertyBase } from './ipropertybase';
import { Photo } from './Photo';

export class Property implements IPropertyBase {
  id: number;
  sellRent: number;
  name: string;
  price: number;
  bhk: number;
  builtArea: number;
  CityId: number;
  city: string;
  image?: string;
  propertyTypeId: number;
  propertyType: string;
  furnishingTypeId: number;
  furnishingType: string;
  carpetArea?: number;
  address: string;
  address2?: string;
  floorNo?: string;
  totalFloors?: string;
  readyToMove: boolean;
  age?: string;
  mainEntrance?: string;
  security?: number;
  gated?: boolean;
  maintenance?: number;
  estPossessionOn?: string;
  description?: string;
  photos?: Photo[];
}

File: bookingSystem/Data/Entities/Property.cs
using System;

namespace Data.Entities
{
    public class Property : BaseEntity
    {
        public string Name { get; set; }
        public string PropertyType { get; set; }
        public string FurnishingType { get; set; }
        public int Price { get; set; }
        public int BuiltArea { get; set; }
        public string Address { get; set; }
        public string Address2 { get; set; }
        public string City { get; set; }
        public string Country { get; set; }
        public int FloorNo { get; set; }
        public int TotalFloors { get; set; }

        public bool ReadyToMove { get; set; }
        public string MainEntrance { get; set; }

        public int Security { get; set; }
        public int Maintenance { get; set; }
        public DateTime DateOfPossession { get; set; }
        public string Description { get; set; }
        public bool Publish { get; set; }
        public string AppUserId { get; set; }
        public AppUser AppUser { get; set; }
    }
}